lms/templates/register-form.html

```diff
@@ -104,6 +104,12 @@ <h2 class="sr">${_('Required Information')}</h2>
       <label for="password">${_('Password')}</label>
       <input id="password" type="password" name="password" value="" required aria-required="true" />
     </li>
+    % if settings.REGISTRATION_EXTRA_FIELDS['password_copy'] == 'required':
```

Comment:
Again, this is the old mako template for the old registration form. Please do not make any more changes to this obsolete area of the code base.